City-of-Helsinki/parking-permits
# Show traficom restrictions

## Context

[PV-680](https://helsinkisolutionoffice.atlassian.net/browse/PV-680)

This adds a `restrictions` field to the Vehicle model, which is parsed in the Traficom API. 

## How Has This Been Tested?

Tested manually

## Manual Testing Instructions for Reviewers

Use this user/vehicle: 

Tenho Testilä (010190-905V) ajoneuvon FIG-403

In webshop, create new permit with this user and vehicle.  There should be a number of restriction codes returned.

## Screenshots

![Screenshot from 2023-12-01 11-36-22](https://github.com/City-of-Helsinki/parking-permits/assets/131681805/642a8490-6f1c-409b-9fdd-7626a06fdf9f)




[PV-680]: https://helsinkisolutionoffice.atlassian.net/browse/PV-680?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
